tailcfg/tailcfg.go

```diff
@@ -1330,9 +1329,9 @@ type MapResponse struct {
 	// ControlTime, if non-zero, is the current timestamp according to the control server.
 	ControlTime *time.Time `json:",omitempty"`
 
-	// TKA, if non-nil, describes updates for the local instance of the
-	// tailnet key authority.
-	TKA *TKAMapResponse `json:",omitempty"`
+	// TKAInstructions, if non-nil, describes updates for the local instance
```

Comment:
The field's named TKA, not TKAInstructions

Reply:
Fixed.